src/sf/providers/twitter.json

```
@@ -6,5 +6,11 @@
       "baseUrl": "https://api.twitter.com"
     }
   ],
+  "parameters": [
+    {
+      "name": "accessToken",
+      "description": "User-authorized access token"
```

This can be misleading as Twitter supports both OAuth 2.0 and OAuth 1.0a authentications. Both authentication specs uses term `Access token`. I would call the parameter `oauth2AccessToken`.

---

This will make usage harder since you will need to explicitly provide a different parameter for Twitter (but it also shows a weakness in this approach). Perhaps we could keep the name `accessToken` and just mention OAuth 2.0 in description?

```suggestion
      "description": "User-authorized OAuth 2.0 access token"
```

---

> This will make usage harder since you will need to explicitly provide a different parameter for Twitter

It is just coincidence that current providers uses the same authentication. Authentication parameters are provider specific.

But I am OK with your solution. Let's just change the description.


<br class="Apple-interchange-newline">